docs/howtos/create_custom_appcharts.md

```diff
@@ -128,30 +154,92 @@ This How-To now simply assumes that the chosen file is named:
 fluentd-appchart.yaml
 ```
 
-> Information: this is a Kubernetes custom resource, which will describe the new Helm
-> chart to Epinio.  In this example, it provides the information about the name, the
-> source and descriptions.
+:::info
+This is a Kubernetes custom resource, which will describe the new Helm chart to Epinio.  In this
+example, it provides the information about the name, the source and descriptions.
+:::
 
 The necessary Kubernetes CRD is provided by the Epinio installation.
 
 Apply this resource to the Epinio cluster:
 
 ```
-$ kubectl apply -f fluentd-appchart.yaml
+kubectl apply -f fluentd-appchart.yaml
 ```
 
 Verify that the new chart is now registered in Epinio:
 
 ```
-$ epinio app chart list
+epinio app chart list
 ```
 
 You can also see the details of the chart in Epinio:
 
 ```
-$ epinio app chart show fluent
+epinio app chart show fluent
```

Comment:
@andreas-kupries, it seems to keep appearing without `d` at the end of `fluent`

Reply:
Now fixed (b0927149bcc81c5d6beb7ba50e48641f4b372d49), had modified a different place